form_jsons/public/funding-application.json

```diff
@@ -241,6 +242,9 @@
         }
       ],
       "next": [
+        {
+          "path": "/dummy-page"
+        }
       ]
```

Comment:
Is there no way to have multiple pages redirecting to the summary (rather than "dummy page" before the summary)? Have we established what the constraints are for page sequences? We should let the design team know if we need to have certain pages in sequence (or otherwise perhaps raise a ticket to configure the runner to not require a "dummy page")

Reply:
+ 1

Reply:
If multiple pages are directly linked to the summary page, it doesn't allow to submit the summary page. it’s protection against a user skipping to the summary page and just clicking submit or we could have four separate summary pages linked to each of the pages from the country list instead of a dummy page which works the same.

Reply:
So somehow we don't need a dummy page anymore. I have updated the JSON :)